build/AzurePipelinesTemplates/WindowsAppSDK-BuildVSIX-Steps.yml

```diff
@@ -0,0 +1,236 @@
+parameters:
+- name: SignOutput
+  type: boolean
+  default: False
+- name: IsOfficial
+  type: boolean
+  default: False 
```

Comment:
Similar question as above.
And IsOfficial seems to be unused?

Reply:
It is being used to determine whether to publish artifacts manually

Reply:
Oh whoops I see where the confusion is. This should be named "IsOneBranch" so that condition line 232 can kick in. Fixing

Reply:
Oh wait no, it should be passed down to to the Publish Symbols yml template.